bokehjs/src/coffee/models/sources/geojson_data_source.coffee

```diff
@@ -18,7 +18,7 @@ class GeoJSONDataSource extends ColumnDataSource.Model
   initialize: (options) ->
     super(options)
     @geojson_to_column_data() # this just validates the initial geojson value
-    @register_property('data', @geojson_to_column_data, true)
+    @define_computed_property('data', @geojson_to_column_data, true)
```

Comment:
Because of this, I can't throw an exception in `define_computed_property()`, just log a warning.